cli/src/pcluster/cli/entrypoint.py

```diff
@@ -215,7 +215,13 @@ def _run_operation(model, args, extra_args):
             error_encoded = encoder.JSONEncoder().encode(message)
             raise APIOperationException(json.loads(error_encoded))
     else:
-        return args.func(args, extra_args)
+        try:
+            return args.func(args, extra_args)
+        except Exception as e:
+            # format exception messages in the same manner as the api
+            message = pcluster.api.errors.exception_message(e)
+            error_encoded = encoder.JSONEncoder().encode(message)
+            raise APIOperationException(json.loads(error_encoded))
```

Comment:
This change may be considered too invasive because it moves the output of some commands to the JSON style provided by the API. It may require then to change integration tests accordingly.

I will move away from this approach and adopt a less invasive one.

Reply:
Currently: the following commands are not defined in the API model, so they would fall into this except block in case of an Exception:

1. configure
2. dcv-connect
3. export-cluster-logs
4. export-image-logs
5. ssh
6. version